kube/harbor-scanner-trivy.yaml

```diff
@@ -13,14 +13,25 @@ spec:
       targetPort: 8443
 ---
 apiVersion: apps/v1
-kind: Deployment
+kind: StatefulSet
 metadata:
   name: harbor-scanner-trivy
 spec:
-  replicas: 1
+  serviceName: harbor-scanner-trivy
   selector:
     matchLabels:
       app: harbor-scanner-trivy
+  replicas: 0
```

Comment:
Does something scale this up?

Reply:
Currently you have to scale up manually with `kubectl`, i.e. `kubect scale sts harbor-scanner-trivy --replicas=1`